app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -421,10 +421,14 @@ function AcuantCapture(
   }
 
   function onAcuantImageCaptureSuccess(nextCapture: AcuantSuccessResponse) {
-    const { image, cardtype, dpi, moire, glare, sharpness } = nextCapture;
+    let { image, cardType, dpi, moire, glare, sharpness } = nextCapture;
+    if (!cardType) {
+      const { cardtype } = nextCapture;
+      cardType = cardtype;
+    }
```

Comment:
what if we pulled out both keys and used like `??` to combine them?
```suggestion
    const { image, cardType: cardType11_9_1, cardtype: cardType11_8_2, dpi, moire, glare, sharpness } = nextCapture;
    const cardType = cardType11_9_1 ?? cardType11_8_2;
```

Reply:
I decided to make a separate type for the older version of acuant, but this is also a good idea!